tests/Pagination/LengthAwarePaginatorTest.php

```diff
@@ -111,4 +114,23 @@ public function testItRetrievesThePaginatorOptions()
     {
         $this->assertSame($this->options, $this->p->getOptions());
     }
+
+    public function testLengthAwarePaginatorCorrectlyJsonSerializesItsItems()
+    {
+        $this->p->setCollection(Collection::make([
+            new class implements Arrayable, JsonSerializable {
+                public function toArray()
+                {
+                    return 'array';
```

Comment:
Please return an actual array here, so the return type is correct. This will matter if we ever add genuine return types to the interfaces in the future.